### ISSUE TITLE
Contrast in sidebar menu

### ISSUE THREAD

**Issue by JenM00re** [jerry-604/swarthmore-a11ygator#4]
The contrast is good when you're scrolling down the page, but in light mode when you mouse over a selection on the sidebar, the text is black with a maroon background. This is not accessible.

Ensure the mousing over on the expandable side menu has a higher contrast / same contrast as the selected folder
